.travis.yml

```diff
@@ -1,9 +1,15 @@
 sudo: false
 language: go
 
-go:
-    - 1.5
-    - 1.6
+matrix:
+    include:
+        - go: 1.5
+          env: LINT=1
+        - go: 1.6
+          env: LINT=1
```

Comment:
You don't want to lint in both 1.5 and 1.6. There are some conflicting rules between the two versions. In thriftrw and yarpc go, we just lint for 1.6.